clayui.com/plugins/gatsby-plugin-svg-compile-sprite/gatsby-node.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: © 2018 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// Compiles SVG icons in directory `clay-css/src/images/icons` into a spritemap
+
+const fs = require('fs');
+const path = require('path');
+
+const generateSvgSprite = (pluginOptions) => {
+	const REGEX_FILE_EXT_SVG = /(flags-|.svg$)/g;
+	const REGEX_HTML_COMMENTS = /<!--(.+?)(-->\n|-->)/gs;
+	const REGEX_SVG_VIEWBOX = /(\sv|v)iewBox="(.+?)"/g;
```

Comment:
Might be easier to write this as: `\bviewBox` (ie. `\b` = word boundary) instead of `(\sv|v)iewBox` and use it like:

```js
const symbol = `<symbol id="${id}" ${viewBox}>`;
```

Like the next line too, `[^"]+` is probably better than the non-greedy group.